yesod-auth/Yesod/Auth/Email.hs

```diff
@@ -346,8 +358,11 @@ authEmail =
 getRegisterR :: YesodAuthEmail master => HandlerT Auth (HandlerT master IO) Html
 getRegisterR = registerHandler
 
-emailLoginHandler :: YesodAuthEmail master => (Route Auth -> Route master) -> WidgetT master IO ()
-emailLoginHandler toParent = do
+-- | Default implementation of 'emailLoginHandler'.
+--
+-- @since 1.2.6
```

Comment:
Not quite sure what to put there thought, the current tag is `1.4.10` but if this is used, it will be on `1.4.11` or the like...